PermissionsEx/permissions.yml

```diff
@@ -2302,6 +2302,25 @@ groups:
         - essentials.skull
         - essentials.skull.*
         - essentials.gamemode.survival
+  doctordoom:
+    inheritance:
+    - mortal
+    options:
+      prefix: '&2D&ao&2ct&ao&2r &aD&2o&ao&2m'
+      suffix: '&b'
+    worlds:
+      survival:
+        permissions:
+        - ch.alias.kill
+        - ch.alias.hoveron
+        - ch.alias.hoveroff
+        - essentials.tp
+        - essentials.tp.*
+        - essentials.gamemode
+        - essentials.gamemode.creative
```

Comment:
that is not matter, that is energy. matter is something you can hold. he is using an energy shield.